argo/providers/vai/provider.go

```diff
@@ -36,8 +33,9 @@ var labels = struct {
 	RunConfigurationNamespace string
 	RunName                   string
 	RunNamespace              string
-	LegacyNamespace           string
-	LegacyRunConfiguration    string
+	// TODO SHOULD WE DELETE THESE NOW?
```

Comment:
@jmendesky

Reply:
comment in code is
// For compatability with resources created with v0.3.0 and older

Reply:
This can be removed now.